app/templates/components/ui-table/cell/events/view/tickets/cell-add-order-total.hbs

```diff
@@ -1 +1 @@
-{{number-format (mult record.orderQuantity record.price)}}
\ No newline at end of file
+{{currency-symbol paymentCurrency (mult record.orderQuantity record.price)}}
```

Comment:
you removed the number-format helper. Why? 
![image](https://user-images.githubusercontent.com/17252805/54870849-11a01a00-4de7-11e9-9e1d-68f0dec18f4b.png)
Now the sub-totals have disappeared. Test locally before pushing.

Reply:
I think it got deleted by mistake , Due to some problem i need to refork this repo will open a new PR with all the changes you said tomorrow :)